src/broadcasting.jl

```diff
@@ -1,23 +1,94 @@
 # See: https://docs.julialang.org/en/v1/manual/interfaces/#man-interfaces-broadcasting-1
 
-Base.BroadcastStyle(::Type{<:NamedDimsArray}) = Broadcast.ArrayStyle{NamedDimsArray}()
+using Base.Broadcast:
+    BroadcastStyle, DefaultArrayStyle, AbstractArrayStyle, Unknown,
+    Broadcasted, broadcasted, materialize
```

Comment:
I am not sure if we want to move these into NamedDims.jl

Reply:
yes, please put all module imports in src/NamedDims.jl :) 
(i understand test imports make sense as file-specifc)